### PR TITLE
Added some good Problems related to DSU

### PR DESCRIPTION
**Pull request by sceary-expert**
These are the problems that one should definitely give a try.
 You might upload solution of these problems (which will be great if you do).
I don't know why but I was not able to make the table. Sorry :(

### CODE REVIEW

**Comment by wingkwong** [comment]
> I don't know why but I was not able to make the table. Sorry :(

Look at arrays.md. I guess you miss this line.

```
import Table from '@site/src/components/Table';
```

**Comment by sceary-expert** [comment]
@wingkwong can you check why it's still not working though I have imported table.

File: tutorials/graph-theory/disjoint-set-union.md [hunk 1]
```diff
@@ -138,4 +139,27 @@ int main() {
 	return 0;
 }
 ```
+export const suggestedProblems = [
+  {
+    "problemName": "Detect Cycle using DSU",
+    "difficulty": "Easy",
+    "leetCodeLink": "https://practice.geeksforgeeks.org/problems/detect-cycle-using-dsu/1",
```
Comment (wingkwong) [inline tutorials/graph-theory/disjoint-set-union.md:146]: Please don't include problems from other sites.

**Review by wingkwong — CHANGES_REQUESTED** [review]

**Comment by wingkwong** [comment]
> @wingkwong can you check why it's still not working though I have imported table.

Are you running locally? Don't look at the Githbu Markdown preview. It's different.

image

**Comment by sceary-expert** [comment]
Yes I was looking at the preview 😅.

**Comment by sceary-expert** [comment]
@wingkwong Is everything alright?

**Review by wingkwong — CHANGES_REQUESTED** [review]
I just imported TutorialAuthors & Table globally. Now these two lines are unnecessary. Please remove it. That should be the conflict.

**Comment by sceary-expert** [comment]
@wingkwong can you check now?

**Review by wingkwong — APPROVED** [review]
LGTM

**Pull request merged**